src/screens/StoryScreen.tsx

```diff
@@ -1,10 +1,42 @@
-import React from 'react';
-import { Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { ActivityIndicator, ScrollView, View } from 'react-native';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import HTMLView from 'react-native-htmlview';
 
 export default function StoryScreen() {
+  const [isLoading, setLoading] = useState(true);
+  const [title, setTitle] = useState(String);
+  const [content, setContent] = useState(String);
+
+  const getStory = async () => {
+    try {
+      const response = await fetch(
+        'https://girlswritenow.org/wp-json/wp/v2/story/170947',
```

Comment:
Let's generalize `getStory` by having it take 1 argument: the ID of a story. When you call the function in the useEffect below, feel free to call it with ID 170947.